### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by RyanBalfanz**
This works, although there are currently no tests.

```
(env) ➜  teambot git:(travis-ci) make test
python -m doctest -v rtmbot.py
19 items had no tests:
    rtmbot
    rtmbot.Plugin
    rtmbot.Plugin.__init__
    rtmbot.Plugin.do
    rtmbot.Plugin.do_output
    rtmbot.RtmBot
    rtmbot.RtmBot.__init__
    rtmbot.RtmBot.autoping
    rtmbot.RtmBot.connect
    rtmbot.RtmBot.input
    rtmbot.RtmBot.load_plugins
    rtmbot.RtmBot.output
    rtmbot.RtmBot.start
    rtmbot.UnknownChannel
    rtmbot.dbg
    rtmbot.flag_is_true
    rtmbot.invoke
    rtmbot.main_loop
    rtmbot.parse_args
0 tests in 19 items.
0 passed and 0 failed.
Test passed.
```

### CODE REVIEW

**Review by tdeck — APPROVED** [review]
LGTM. I think doctest probably won't be the most fruitful option in the future, but we'll see. I always thought it was cool :).

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 teambot
 =======
+
+[![Build Status](https://travis-ci.org/RyanBalfanz/teambot.svg?branch=settings-module-config)](https://travis-ci.org/RyanBalfanz/teambot)
```
Comment (RyanBalfanz) [inline README.md:4]: This will need to be changed to https://github.com/tdeck/teambot.

Comment (tdeck) [reply README.md:4]: LGTM. I created a project for it.

Comment (RyanBalfanz) [reply README.md:4]: Haha, yeah I was on Travis and I saw it start. I was wondering how that happened… until I came back here. 👍 

screen shot 2017-03-29 at 9 33 37 pm

Comment (RyanBalfanz) [reply README.md:4]: @tdeck Go ahead and update the README to point back at your project.

**Pull request merged**